README.md

```diff
@@ -521,26 +439,126 @@ with self.dag:
     )
 ```
 
-## Raw SQL
-Most ETL use-cases can be addressed by cross-sharing Task outputs, as shown above with `@aql.transform`. For SQL operations that _don't_ return tables but might take tables as arguments, there is `@aql.run_raw_sql`. 
+### Raw SQL
+
+Most ETL use cases can be addressed by cross-sharing task outputs, as shown above with `@aql.transform`. If you need to perform SQL operation that doesn't return a table but might take a table as an argument, you can use `@aql.run_raw_sql`. 
 
 ```python
 @aql.run_raw_sql
 def drop_table(table_to_drop):
     return "DROP TABLE IF EXISTS {table_to_drop}"
 ```
 
+## Putting it All Together 
+
+The following is a full example DAG of a SQL + Python workflow using `astro`.
```

Comment:
```suggestion
The following is a full example DAG of a SQL + Python workflow using `astro`. We pull data from S3, run SQL transformations to merge our pulled data with existing data, and move the result of that merge into a dataframe so that we can complete complex work on it using Python / ML.
```